src/main/java/programminglife/parser/GraphParser.java

```diff
@@ -113,8 +117,13 @@ protected synchronized void parse(boolean verbose) throws FileNotFoundException,
                     default:
                         throw new UnknownTypeException(String.format("Unknown symbol '%c'", type));
                 }
-
-                if (progressCounter.getLineCount() % (1000 * (this.isCached ? 100 : 1)) == 0) {
+                int tmp;
+                if (isCached) {
+                    tmp = 100;
+                } else {
+                    tmp = 1;
+                }
+                if (progressCounter.getLineCount() % (1000 * tmp) == 0) {
```

Comment:
Maybe use a more descriptive name than `tmp`, like `lineSkipFactor`.
Also, you could just use 100.000 and 1.000 and then do `[...] % tmp`